app/helpers/patient_query_helper.rb

```diff
@@ -2,6 +2,33 @@
 
 # Helper methods for filtering through patients
 module PatientQueryHelper # rubocop:todo Metrics/ModuleLength
+  def patients_table_data(params)
```

Comment:
Any strong reason for not keeping this directly on the controller? In my opinion it's less of a "query" than a "query gatherer". In particular it's doing all sorts of pagination stuff - which would be better to see in the controller. Not strongly attached to that idea, but worth hearing your thoughts.

Reply:
Yes, because we need to be able to access it from different routes outside of that controller. I suppose we could still extract it as a separate function on that route and call it outside of an action, but since it's no longer public health user role specific, it felt wrong to keep it in the public_health_controller.

Reply:
> need to be able to access it from different routes outside of that controller
Good enough for me!